dockerfiles/gomoderator/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:alpine as builder
+
+RUN apk --update --no-cache add make git g++ linux-headers python3
+
+ARG VDB_VERSION=staging
+ENV GO111MODULE on
+
+WORKDIR /go/src/github.com/makerdao
+RUN git clone https://github.com/makerdao/vulcanizedb.git
+RUN git clone https://github.com/makerdao/vdb-mcd-transformers.git
```

Comment:
I don't think we need to clone this repo (and probably don't want to so we avoid being on the wrong branch).

Instead maybe we can just `COPY . .` (may want to enter the appropriate workdir first)